src/main/java/tools/image/DecorationPlacer.java

```diff
@@ -265,7 +265,8 @@ public void mouseMoved(final MouseEvent e) {
     imagePanel.addMouseListener(new MouseAdapter() {
       @Override
       public void mouseClicked(final MouseEvent e) {
-        mouseEvent(e.getPoint(), e.isControlDown() || e.isShiftDown(), SwingUtilities.isRightMouseButton(e));
+        final Point point = e.getPoint();
```

Comment:
Added unused local variable `point` to preserve any possible side effects caused by the call to `e.getPoint()`.